pkg/models/html.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html/template"
+	"io"
+
+	"github.com/cmsgov/easi-app/pkg/sanitization"
+)
+
+// HTML represents html code. It is sanitized to only allow specific tags
+type HTML string
+
+// ToTemplate converts the HTML type to a template.HTML struct
+func (html HTML) ToTemplate() template.HTML {
+	// Note, we sanitize when the data is Unmarshaled. However, if desired it could be sanitized at this entrypoint as well
+	return template.HTML(html) //nolint //the html is sanitized when it is Unmarshaled
+}
+
+// ValueOrEmptyString returns either the value of the html or an empty string if nil
+func (html *HTML) ValueOrEmptyString() string {
+	if html != nil {
+		return string(*html)
+	}
+	return ""
+}
+
+// ValueOrEmptyHTML returns either the value of the html or an empty HTML type if nil
+func (html *HTML) ValueOrEmptyHTML() HTML {
+	if html != nil {
+		return *html
+	}
+	return ""
+}
+
+// StringPointer casts an HTML pointer to a string pointer
+func (html *HTML) StringPointer() *string {
+	return (*string)(html)
+}
+
+// UnmarshalGQLContext unmarshals the data from graphql to the HTML type
+func (html *HTML) UnmarshalGQLContext(ctx context.Context, v interface{}) error {
+
+	// Check if the value is a string
+	htmlString, ok := v.(string)
+	if !ok {
+		return errors.New("invalid HTML")
+	}
+
+	// Sanitize the HTML string
+	sanitizedHTMLString := sanitization.SanitizeHTML(htmlString)
+
+	// Set the sanitized HTML value to the receiver
+	*html = HTML(sanitizedHTMLString)
+
+	return nil
+}
+
+// MarshalGQLContext marshals the HTML type to JSON to return to graphQL
+func (html HTML) MarshalGQLContext(ctx context.Context, w io.Writer) error {
```

Comment:
Similar to unmarshaling, it doesn't look this uses the context - could you use [`MarshalGQL`](https://pkg.go.dev/github.com/99designs/gqlgen/graphql#Marshaler)?